Plugin NPC/src/main/java/net/simon987/npcplugin/HarvesterNPC.java

```diff
@@ -36,6 +41,36 @@ public void update() {
         }
     }
 
+    @Override
+    public void setDead(boolean dead) {
+        super.setDead(dead);
+//        GameServer.INSTANCE.getEventDispatcher().dispatch(
+//                new ObjectDeathEvent((Object)this, ID));
+    }
+
+    @Override
+    public void onDeadCallback() {
+        GameServer.INSTANCE.getEventDispatcher().dispatch(
+            new ObjectDeathEvent((Object)this, ID));
+    }
+
+    public BiomassBlob createBiomassBlobInPlace() {
```

Comment:
I would prefer that the createBiomassBlobInPlace() function would be inside the Plant Plugin's ObjectDeathListener